### PR TITLE
chaosd document: fix config && add clock for server mode

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

1. update some config, since their field is changed in https://github.com/chaos-mesh/chaosd/pull/100
2. add clock for server mode

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-disk-pressure-in-physical-nodes.md [hunk 1]
```diff
@@ -256,15 +256,15 @@ curl -X POST 172.16.112.130:31767/api/attack/disk -H "Content-Type:application/j
 | :-------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | action | 实验的行为 | 设置为 "fill" |
 | destroy   | 如果此参数为 true ，则在填充文件后立即删除填充文件              | bool 类型，默认为 false。        |
-| fill_by_fallocate | 如果此参数为 true ，则使用 linux 调用 fallocate 来快速申请磁盘空间，此时 size 必须大于 0。如果此参数为 false，则使用 linux 调用 dd 以相对较慢速度填充磁盘。              | bool 类型，默认为 true。                                                |
+| fill-by-fallocate | 如果此参数为 true ，则使用 linux 调用 fallocate 来快速申请磁盘空间，此时 size 必须大于 0。如果此参数为 false，则使用 linux 调用 dd 以相对较慢速度填充磁盘。              | bool 类型，默认为 true。                                                |
 | path     | 指定所写数据的文件路径。如果没有设置此参数，或者设置参数值为空字符串，则会在程序执行目录下创建一个临时文件。根据写入文件的权限不同，会需要你使用一定的权限运行本程序。 | string 类型，默认为 ""       |
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-disk-pressure-in-physical-nodes.md:260]:
```suggestion
| path     | 指定所写数据的文件路径。如果没有设置此参数，或者设置参数值为空字符串，则会在程序执行目录下创建一个临时文件。根据写入文件的权限不同，会需要你使用一定的权限运行本程序。 | string 类型，默认为 ""。       |  
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-disk-pressure-in-physical-nodes.md [hunk 2]
```diff
@@ -256,15 +256,15 @@ curl -X POST 172.16.112.130:31767/api/attack/disk -H "Content-Type:application/j
...
 | action | 实验的行为 | 设置为 "fill" |
 | destroy   | 如果此参数为 true ，则在填充文件后立即删除填充文件              | bool 类型，默认为 false。        |
-| fill_by_fallocate | 如果此参数为 true ，则使用 linux 调用 fallocate 来快速申请磁盘空间，此时 size 必须大于 0。如果此参数为 false，则使用 linux 调用 dd 以相对较慢速度填充磁盘。              | bool 类型，默认为 true。                                                |
+| fill-by-fallocate | 如果此参数为 true ，则使用 linux 调用 fallocate 来快速申请磁盘空间，此时 size 必须大于 0。如果此参数为 false，则使用 linux 调用 dd 以相对较慢速度填充磁盘。              | bool 类型，默认为 true。                                                |
 | path     | 指定所写数据的文件路径。如果没有设置此参数，或者设置参数值为空字符串，则会在程序执行目录下创建一个临时文件。根据写入文件的权限不同，会需要你使用一定的权限运行本程序。 | string 类型，默认为 ""       |  
 | percent   | 指定填充多少百分比磁盘。                                             | string 类型，默认为 ""，可以填入 uint 类型的正整数，size 和 percent 不能同时为 ""     |
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-disk-pressure-in-physical-nodes.md:261]:
```suggestion
| percent   | 指定填充多少百分比磁盘。                                             | string 类型，默认为 ""，可以填入 uint 类型的正整数，size 和 percent 不能同时为 ""。     |
```

**Review by cwen0 — APPROVED** [review]
LGTM

**Comment by cwen0** [comment]
@en-jin19 PTAL

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md [hunk 3]
```diff
@@ -75,6 +75,20 @@ chaosd attack clock -p $PID -t 11s
...
 
-### 使用服务模式创建实验
+## 使用服务模式创建实验
 
-（正在持续更新中）
+### 模拟时间故障相关参数说明
+
+| 参数      | 说明    | 值       |
+| :---------- | :----------------------------------------------------------------- | :-------------------------------------------------------------------------- |
+| pid | 进程的标识符。 | int 类型。 |
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md:84]:
```suggestion
| pid | 进程的标识符。 | int 类型 |
```

**Review by en-jin19 — APPROVED** [review]
rest LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md (hunk 3 above)
Comment (WangXiangUSTC) [reply i18n/zh/docusaurus-plugin-content-docs/current/simulate-time-chaos-on-physical-nodes.md]: done

**Comment by WangXiangUSTC** [comment]
@en-jin19 PTAL again

**Pull request merged**